mscore/editstyle.cpp

```diff
@@ -970,5 +972,39 @@ void EditStyle::resetStyleValue(int i)
       setValues();
       }
 
+//---------------------------------------------------------
+//   closeEvent
+//---------------------------------------------------------
+
+void EditStyle::closeEvent( QCloseEvent* )
+      {
+      writeSettings();
+      }
+
+//---------------------------------------------------------
+//   writeSettings
+//---------------------------------------------------------
+
+void EditStyle::writeSettings()
+      {
+      QSettings settings( "museScore", "editStyle" );
```

Comment:
ditto and on most of the following lines too. Drop the space after ( and before )